bin/hubot

```diff
@@ -3,9 +3,12 @@
 set -e
 
 fail() {
-    echo "$1"
-    echo "See README.md for more information"
-    exit 1
+   echo "$1"
```

Comment:
Looks like indentation is now `3` spaces for this code block. Not sure if you really wanted that ;)

Reply:
fixed.